common/cliutils/cli_consts.go

```diff
@@ -1,13 +1,12 @@
 package cliutils
 
-import ()
-
 type CommandDomain string
 
 const (
 	Rt       CommandDomain = "rt"
 	Ds       CommandDomain = "ds"
 	Xr       CommandDomain = "xr"
+	Sbom     CommandDomain = "sbom"
```

Comment:
I don't think that adding a new "sbom" sub-command is the way to go.
Our general approach is pretty much the opposite.